src/Features/Vendor/BalanceSheet/index.js

```diff
@@ -11,6 +11,7 @@ import { FetchBalances } from './requests';
 
 const BalanceSheet = () => {
   const { isLoading, data, refetch } = FetchBalances();
+  const [pageName] = useState('BalanceSheet');
```

Comment:
why do you create a variable for just passing the props? While you can directly pass string as prop

Reply:
shahir bhai direct string pass kera dia as a prop.